src/main/kotlin/no/nav/helse/spenn/oppdrag/OppdragMapper.kt

```diff
@@ -178,3 +179,14 @@ private fun mapTolinje150(oppdragslinje : UtbetalingsLinje, maksDato: LocalDate)
 
     }
 }
+
+fun UUID.toFagId(): String {
+    val byteBuf = ByteBuffer.wrap(ByteArray(16)).putLong(mostSignificantBits).putLong(leastSignificantBits)
+    val encodeToString = Base64.getUrlEncoder().encodeToString(byteBuf.array())
+    return encodeToString.substring(0, encodeToString.length-2)
+}
+
+fun String.toUUID(): UUID {
```

Comment:
Potensielt meeeget forvirrende misbruk av extension-methods for de som ikke kjenner historien, ja? ;-)    (siden den er på "String" og ikke sier hva den konverterer fra)

Reply:
yepp, helt enig. Skal fikse.